### PR TITLE
 Allow staging image testing before release

### PR DESCRIPTION
**Pull request by jasonwzm**

### CODE REVIEW

File: asm/istio/expansion/vm-eastwest-gateway.yaml [hunk 1]
```diff
@@ -45,3 +47,8 @@ spec:
               - name: tls-webhook
                 port: 15017
                 targetPort: 15017
+  values:
+    gateways:
+      istio-ingressgateway:
+        # Enable gateway injection
+        injectionTemplate: gateway
```
Comment (zerobfd) [inline asm/istio/expansion/vm-eastwest-gateway.yaml:54]: This doesn't look related to staging images, was this included on purpose?

Comment (jasonwzm) [reply asm/istio/expansion/vm-eastwest-gateway.yaml:54]: Yes. This is added to use the injection based gateway so that 1.10 eastwest gateway can be created.

**Review by zerobfd — APPROVED** [review]

**Review by maasen — APPROVED** [review]
LGTM
